### DISCUSSION TITLE
Socket.io listening state

### DISCUSSION THREAD

**Discussion by mcmaclachlan** [artilleryio/artillery#989] [Artillery]
Hi!
Once a number of clients are connected to a socket.io server, how can they then be set to listen and receive a broadcast from the server?
Thanks in advance for any help!

**Comment by hassy** [comment]
hi @mcmaclachlan 👋 what would you like to be able to do with the messages broadcast from the server?

**Reply by mcmaclachlan** [reply]
hi @hassy, the application will have a teacher plus a number of students connected to the server. The teacher will broadcast to the students when a test is ready sending the questions (json); I would like to be able to confirm that the student users receive the broadcast (questions).  Thanks in advance!

**Comment by mcmaclachlan** [comment]
Hi again @hassy, please disregard my previous comment. The broadcast
actually just notifies the students that an test is ready, it doesn’t send
the test data. The application does have a number of listeners set up right
off the bat to though, so it would be helpful to know how to deal with this
in artillery and confirm receipt of messages. Thanks again!
